apps/redi-talent-pool/src/components/organisms/jobseeker-profile-editables/EditableEducation.tsx

```diff
@@ -97,9 +98,13 @@ export function EditableEducation({
                 </span>
               </div>
               <Content style={{ marginTop: '-0.5rem' }}>
-                {item.institutionName ? (
-                  <p style={{ color: '#979797' }}>{item.institutionName}</p>
-                ) : null}
+                <p style={{ color: '#979797' }}>
+                  {locationDetails(
+                    item?.institutionName,
+                    item?.institutionCity,
+                    item?.institutionCountry
+                  )}
+                </p>
```

Comment:
Same comment with here: https://github.com/talent-connect/connect/pull/465/files#r760448535

Sorry I saw that before this so I first posted there.

Reply:
I assume that the two comments above are not valid anymore since we decided to have ´instituteName´ and ´company´ set to ´required´?